<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bool1</class>
 <widget class="QMainWindow" name="bool1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2051</width>
    <height>1003</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>逻辑运算 V1.0.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>391</width>
      <height>741</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>与门</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>或门</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>非门</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>异或门</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>同或门</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>941</width>
      <height>891</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1031</width>
        <height>561</height>
       </rect>
      </property>
      <property name="text">
       <string>p1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>160</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QLabel" name="and0">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>510</y>
        <width>301</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="ll">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>560</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>与门</string>
      </property>
     </widget>
     <widget class="QLabel" name="and1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>480</y>
        <width>261</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="orpic">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1131</width>
        <height>561</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>130</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>280</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QLabel" name="or1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>480</y>
        <width>281</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="or0">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>500</y>
        <width>311</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>570</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>或门</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="no1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>460</y>
        <width>281</width>
        <height>261</height>
       </rect>
      </property>
      <property name="text">
       <string>p3</string>
      </property>
     </widget>
     <widget class="QLabel" name="no0">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>500</y>
        <width>261</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>570</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>非门</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>250</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QLabel" name="nopic">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>1051</width>
        <height>481</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <zorder>nopic</zorder>
     <zorder>no1</zorder>
     <zorder>no0</zorder>
     <zorder>label_3</zorder>
     <zorder>pushButton_10</zorder>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="norpic">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-30</y>
        <width>941</width>
        <height>611</height>
       </rect>
      </property>
      <property name="text">
       <string>p4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QLabel" name="input2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Input2:0</string>
      </property>
     </widget>
     <widget class="QLabel" name="input1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Input1:0</string>
      </property>
     </widget>
     <widget class="QLabel" name="pd1">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>170</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pd2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>330</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pd3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>330</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="output">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>230</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Output:0</string>
      </property>
     </widget>
     <widget class="QLabel" name="nornum">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>330</y>
        <width>251</width>
        <height>451</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="nor">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>490</y>
        <width>291</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>540</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>异或门</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="xnorpic">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>941</width>
        <height>441</height>
       </rect>
      </property>
      <property name="text">
       <string>p5</string>
      </property>
     </widget>
     <widget class="QLabel" name="xnornum">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>450</y>
        <width>241</width>
        <height>251</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="xnor">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>520</y>
        <width>301</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>560</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>同或门</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>350</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QLabel" name="pd4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>210</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pd5">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>110</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pd6">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>300</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pd7">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>210</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>pd7</string>
      </property>
     </widget>
     <widget class="QLabel" name="xoutput">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>170</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Output:1</string>
      </property>
     </widget>
     <widget class="QLabel" name="xinput1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>input1:0</string>
      </property>
     </widget>
     <widget class="QLabel" name="xinput2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>input2:0</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
